pru/prudebug/prudbg.h

```diff
@@ -10,14 +10,15 @@
 #define PRUDBG_H
 
 // default processor to use if none is specified on the command line when prudebug is started
-#define DEFAULT_PROCESSOR_INDEX	AM335x
+#define DEFAULT_PROCESSOR_INDEX	AM57x1
```

Comment:
why change the default processor index? Is BBAI more common than BBB and PocketBeagle?

Reply:
>why change the default processor index? Is BBAI more common than BBB and PocketBeagle?

Okay, I will revert this.